### PR TITLE
Upgrades all dependencies

### PR DESCRIPTION
**Pull request by pellebjerkestrand**
This PR upgrades all dependencies (and removes unused ones).

It also removes Chimp, as it's not being maintained anymore, and replaces it with TestCafe. TestCafe seems OK in use.

Attempts to solve #35

### CODE REVIEW

File: static-files/webpack.config.js [hunk 1]
```diff
@@ -50,14 +50,9 @@ module.exports = (env = {}, options = {}) => {
 
       // NOTE: when using mockup with 'yarn dev', only static.js will be served
       if (shouldBuildStaticSite) {
-        entries.static = [
-          'babel-polyfill',
-          'whatwg-fetch',
-          './source/static.js'
-        ];
+        entries.static = ['whatwg-fetch', './source/static.js'];
```
Comment (asbjornh) [inline static-files/webpack.config.js]: Why no polyfill?

File: README.md [hunk 2]
```diff
@@ -177,29 +177,14 @@ These aliases are also included in `jsconfig.json` which makes VS Code resolve t
...
-* `.no-touchevents`: The last event was not a touch event
+- `.mouse-user`: The last event was either a mouse or touch event (this is removed when a keyboard event occurs)
+- `.touchevents`: The last event was a touch event
+- `.no-touchevents`: The last event was not a touch event
 
 By default, all focus outlines are disabled when the `.mouse-user` class is present (in `site.scss`). No class names are present if js is disabled in the browser, meaning focus outlines are preserved.
 
 You can use these classnames to provide alternative styling based on input method (like disabling hover effects for touch screens).
 
-## UI Testing
```
Comment (asbjornh) [inline README.md]: Don't we need any documentation about this at all? I think we at least need to have something about the fact that you need to start webpack dev server in order for tests to work, which isn't obvious.

Comment (pellebjerkestrand) [reply README.md]: I forgot to add it in again 😞 Will do.

File: static-files/webpack.config.js (hunk 1 above)
Comment (pellebjerkestrand) [reply static-files/webpack.config.js]: There is, but it's based on usage. `"useBuiltIns": "usage"`.

**Comment by pellebjerkestrand** [comment]
Upgraded everything to latest again. Anyone interested in merging this?

**Comment by asbjornh** [comment]
@pellebjerkestrand I'm interested once all comments have been addressed :)

**Comment by pellebjerkestrand** [comment]
Which comments haven't been addressed, @asbjornh?

**Comment by asbjornh** [comment]
The one on package.json. I couldn't get a link for it

**Pull request merged**